index.html

```diff
@@ -309,11 +309,20 @@
     <section class="informative">
       <h2>Introduction</h2>
       <p>
-A [=controller document=] is a set of data that specifies one or more
-relationships between a [=controller=] and a set of data, such as a set of
-public cryptographic keys. The [=controller document=] contains [=verification
-relationships=] that explicitly permit the use of certain [=verification
-methods=] for specific purposes.
+        [=Controller Documents=] enable the verification of proofs created
+        by the controller of an identifier. They provide public
+        cryptographic material, e.g., public keys, known as [=verification
+        methods=], for verifying proofs created by the controller of the
```

Comment:
I would reverse the emphasis here to say verification methods are provided and then give an example:

```suggestion
        by the controller of an identifier. They provide [=verification
        methods=] that express public cryptographic material, such as
        public keys, for verifying proofs created by the controller of the
```

This feels more accurately flexible and doesn't necessarily say that verification methods are *equivalent to* public keys, but rather that they can express them (and also doesn't perhaps imply that they can't express more than public key in a single verification method, and so on).

Reply:
Happy to accept. Unfortunately, taking Manu's suggestion killed your edit.

Can you recomment?

Reply:
Edit incoming.